### PR TITLE
feat: AddRange for markers

### PR DESCRIPTION
**Pull request by gerhardol**

### CODE REVIEW

File: Project/Src/Document/MarkerStrategy/MarkerStrategy.cs [hunk 1]
```diff
@@ -34,6 +34,12 @@ public void AddMarker(TextMarker item)
             textMarker.Add(item);
         }
 
+        public void AddMarkerRange(IList<TextMarker> items)
```
Comment (mstv) [inline Project/Src/Document/MarkerStrategy/MarkerStrategy.cs]:
```suggestion
        public void AddMarkers(IEnumerable<TextMarker> items)
```

Comment (gerhardol) [reply Project/Src/Document/MarkerStrategy/MarkerStrategy.cs]: It is AddRange though...

Comment (mstv) [reply Project/Src/Document/MarkerStrategy/MarkerStrategy.cs]: > It is AddRange though...

I have no very strong opinion about the name. Though it does not matter how it is implemented. "Range" makes me think about whether it is sub-range of the collection.

My main point is the more generic argument type.

**Review by mstv — APPROVED** [review]

File: Project/Src/Document/MarkerStrategy/MarkerStrategy.cs (hunk 1 above)
Comment (mstv) [reply Project/Src/Document/MarkerStrategy/MarkerStrategy.cs]: Yesterday when approving, I had not noticed the missing plural "s".

Comment (gerhardol) [reply Project/Src/Document/MarkerStrategy/MarkerStrategy.cs]: Do you want a new PR?

Comment (mstv) [reply Project/Src/Document/MarkerStrategy/MarkerStrategy.cs]: Igor uses to have opinions on names, too. So let's wait.

File: Project/Src/Document/MarkerStrategy/MarkerStrategy.cs [hunk 2]
```diff
@@ -34,6 +34,12 @@ public void AddMarker(TextMarker item)
             textMarker.Add(item);
         }
 
+        public void AddMarker(IList<TextMarker> items)
```
Comment (RussKie) [inline Project/Src/Document/MarkerStrategy/MarkerStrategy.cs:37]: Should it be pluralised?

Comment (mstv) [reply Project/Src/Document/MarkerStrategy/MarkerStrategy.cs:37]: >> Yesterday when approving, I had not noticed the missing plural "s".

and I missed that only the name was changed. 

>> Igor uses to have opinions on names, too. So let's wait.

> Should it be pluralised?

Yes, #41

**Pull request merged**